### PR TITLE
Invoker health ping through kafka

### PR DESCRIPTION
**Pull request by markusthoemmes**
Phase 1 of the new invoker health. No semantic change intended so far.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
PG1 1175

File: core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ */
+class InvokerActor extends FSM[InvokerState, InvokerInfo] {
+    implicit val transid = TransactionId.invokerHealth
+    val logging = new AkkaLogging(context.system.log)
+    def name = self.path.name
+
+    val healthyTimeout = 20.seconds
+
+    // Concat the original receive with a GetStatus event
+    // This is done at this point to not intermangle with the statemachine
```
Comment (perryibm) [inline core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala]: intermingle with the state machine

**Comment by perryibm** [comment]
20 seconds / 2 seconds means 10 missing pings to be considered offline?  Given the granularity, perhaps we could make it more like 10 seconds / 1 seconds to tighten things.   Is it the case that it takes only a single ping to restore it?

**Comment by markusthoemmes** [comment]
@perryibm yep, sure we can. Yes, a single ping will restore it at this point.

**Comment by markusthoemmes** [comment]
@perryibm addressed your comments.

**Comment by rabbah** [comment]
@perryibm deferring to you to merge.

**Comment by perryibm** [comment]
merged

**Pull request closed without merging**